canopy/design-system/src/views/Introduction.mdx

```diff
@@ -0,0 +1,16 @@
+<!-- Copyright 2019 Cargill Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+# Introduction
+
+This styleguide acts as a home for Canopy brand guidelines, styling guidance and helpers, and UI components. Sapling creators are encouraged to use these suggestions and tools while building their Saplings to create a smooth user experience.
```

Comment:
Wrap lines at 80 cols

Reply:
updated

Reply:
It looks like you fixed it in the wrong commit